app/code/Magento/Directory/Model/Currency/Import/CurrencyConverterApi.php

```diff
@@ -75,10 +80,15 @@ public function fetchRates()
     private function convertBatch($data, $currencyFrom, $currenciesTo)
     {
         foreach ($currenciesTo as $to) {
+            //phpcs:ignore Magento2.Functions.DiscouragedFunction
             set_time_limit(0);
             try {
                 $url = str_replace('{{CURRENCY_FROM}}', $currencyFrom, self::CURRENCY_CONVERTER_URL);
                 $url = str_replace('{{CURRENCY_TO}}', $to, $url);
+                $url = $url . '&apiKey=' . $this->scopeConfig->getValue(
```

Comment:
Could you add 'apiKey' parameter to the constant and replace it with with the value?

Reply:
I added it. Please check it again. Thanks.